components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -3606,4 +3630,30 @@
                 "   UUID = ? AND TENANT_ID = ?";
 
     }
+
+    public static final String GET_PENDING_REGISTRATIONS_FOR_APPLICATION_LIST =
+            "SELECT KEY_MANAGER FROM AM_APPLICATION_KEY_MAPPING WHERE " + "APPLICATION_ID IN (_APPLICATION_IDS_) "
+                    + "AND KEY_TYPE = ? AND STATE = 'CREATED'";
```

Comment:
Without passing _APPLICATION_IDS_, shall we modify the query to pass the org ID?